openwisp_notifications/templates/admin/change_form.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'admin/change_form.html' %}
+
+{% block admin_change_form_document_ready %}
+    {{ block.super }}
```

Comment:
I think we need to add a comment (using the django template comment syntax `{# comment #} that explains what this template does)